test/src/test_utils.h

```diff
@@ -5,33 +5,67 @@
  */
 #ifndef TIMESCALEDB_TEST_UTILS_H
 #define TIMESCALEDB_TEST_UTILS_H
-#include <postgres.h>
 
+#include <postgres.h>
 #include <access/xact.h>
 
-#define AssertInt64Eq(a, b)                                                                        \
+#include "export.h"
+
+static inline const char *
+strip_path(const char *filename)
+{
+	int i = 0, slash = 0;
+
+	while (filename[i] != '\0')
+	{
+		if (filename[i] == '/' || filename[i] == '\\')
+			slash = i;
+		i++;
+	}
+
+	return &filename[slash + 1];
+}
```

Comment:
Sad that there is no `strrpbrk` or I would have suggested that.

Reply:
This needs to work also on Windows, otherwise I'd used dirname() or something similar.